apps/console/src/features/roles/components/wizard-updated/role-basics.tsx

```diff
@@ -93,12 +93,18 @@ export const RoleBasics: FunctionComponent<RoleBasicProps> = (props: RoleBasicPr
 
     const noApplicationsAvailable: MutableRefObject<boolean> = useRef<boolean>(false);
 
+    /**
+     * Index of the roles tab.
+     */
+    const ROLES_TAB_INDEX: number = 5;
+
+
     const {
         data: applicationList,
         isLoading: isApplicationListFetchRequestLoading,
         error: applicationListFetchRequestError,
         mutate: mutateApplicationListFetchRequest
-    } = useApplicationList("clientId", null, null, applicationSearchQuery);
+    } = useApplicationList("clientId,associatedRoles.allowedAudience", null, null, applicationSearchQuery);
```

Comment:
```suggestion
    } = useApplicationList("clientId, associatedRoles.allowedAudience", null, null, applicationSearchQuery);
```

Reply:
These are the attributes we are passing in the API request URL. Adding space here makes the request invalid.